suitcase/tiff/conftest.py

```diff
@@ -1,4 +1 @@
-from bluesky.tests.conftest import RE  # noqa
```

Comment:
At least when running the tests locally we need to keep each of these imports. otherwise I get the error:
```
_____________________ ERROR at setup of test_export[True] ______________________
file /home/alwalter/NSLS-II/suitcase-tiff/suitcase/tiff/tests.py, line 13
  @pytest.mark.parametrize('stack_images', [True, False])
  def test_export(tmp_path, example_data, stack_images):
file /home/alwalter/NSLS-II/suitcase-utils/suitcase/utils/tests/conftest.py, line 168
  @pytest.fixture
  def example_data(generate_data, plan_type):
E       fixture 'generate_data' not found
>       available fixtures: benchmark, benchmark_weave, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, example_data, monkeypatch, pytestconfig, record_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/alwalter/NSLS-II/suitcase-utils/suitcase/utils/tests/conftest.py:168

```